### PR TITLE
Update list of allowed images on prod

### PR DESCRIPTION
**Pull request by ashb**
We need this now as we have customers needing to use a debian image, and
the current list on prod does not "allow" this. (It's a warning the user
can ignore, but users should not see a warning for an image we are
recommending the use to work around a bug.)

This list was taken from https://github.com/astronomer/astronomer/commit/2a1b19aa09234f3a4c9fd90406038564b011bbd7

### CODE REVIEW

File: prod/cloud/app/environment [hunk 1]
```diff
@@ -1 +1 @@
-ASTRO_VERSION="0.15.0-rc.1"
+ASTRO_VERSION="0.14.0-rc.1"
```
Comment (sjmiller609) [inline prod/cloud/app/environment:1]: revert this change please, we are trying to ship 0.15 to prod in the near term

Comment (sjmiller609) [inline prod/cloud/app/environment:1]:
```suggestion
ASTRO_VERSION="0.15.0-rc.1"
```

Comment (ashb) [reply prod/cloud/app/environment]: How near? Our customers are getting a bad experience right now.

A happening to staging should mean we can't deploy to staging.
